### DISCUSSION TITLE
Error validating against schema - Invalid content was found

### DISCUSSION THREAD

**Discussion by OscarD31** [INSPIRE-MIF/helpdesk-validator#957] [Q&A] [answered]
Good morning.

[BU_2.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/11462050/BU_2.zip)

I'm validating a GML for the theme Buildings and I keep getting this error:

`XML document 'BU_2.gml': 9:59: cvc-complex-type.2.4.a: Invalid content was found starting with element '{"http://inspire.ec.europa.eu/schemas/bu-core2d/4.0":Building}'. One of '{"http://www.opengis.net/gml/3.2":AbstractFeature}' is expected.`

I don't understand why it is expecting an "AbstractFeature" element instead of a "Building" element.

Does anybody know what I'm doing wrong? I've attached the GML file.

Thanks for any help provided.

**Comment by fabiovinci** [comment]
Dear @OscarD31,

the error should be related to the temporary unavailability of the schema.
I had a look at your gml and it is valid, in fact, it passes the test "Validation against INSPIRE official schema".

![image](https://github.com/INSPIRE-MIF/helpdesk-validator/assets/22212406/4a69745f-ab28-4bb2-8b3e-03ea966f8e31)

**Reply by OscarD31** [reply]
Thanks for the answer! I'm glad it's a temporary issue.

**Reply by fabiovinci** [reply, marked as answer]
Dear @OscarD31,

sorry, I need to correct myself, the error is due to the incorrect declaration of some namespaces in the schemaLocation attribute.

You should use the URLs in http, i.e. _http://inspire.ec.europa.eu/schemas/bu-base/4.0_ and _http://inspire.ec.europa.eu/schemas/bu-core2d/4.0_.
With this fix the gml validates, see [the report](https://inspire.ec.europa.eu/validator/test-run/details.html?id=EID563de5a2-275f-41bf-b0ba-73f917855ee9).

**Reply by OscarD31** [reply]
Thanks!

I forgot about the https on the namespaces. I removed the ones from the system reference URI or the href from control lists, but not those.

Thanks for checking again and cacthing the issue! :)
